scripts/build/chipbuild.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env -S python3 -B
```

Comment:
Why?

Reply:
I did  not want to generate __pycache__ files because I was running scripts in place.
Do we care?

Reply:
I think you should live with it or deal with it via an environment variable rather than special case this.